### ISSUE TITLE
Release: Sprint 15 - 6/22/2016

### ISSUE THREAD

**Issue by noahmanger** [18F/FEC#360]
## Documentation
- [x] Write simplified release notes for FEC with instructions for review
- [x] Update documentation in any relevant repo

### Communciations
- [x] Are there communications around this release from 18F or FEC?

## Reviews
- [x] 18F QA review on staging
- [x] FEC QA review on staging
- [x] File accessibility review request [here](https://github.com/18F/Accessibility_Reviews/issues/new)
- [x] Verify all issues and pull requests marked "before release" are resolved

## Deployment
- [x] Update fec-style and publish to npm as necessary
- [x] Run update scripts in standalone app if necessary
  - [x] Allocate additional resources for the database to help with the processing before starting if need be.
- [x] Make sure local copies of `master`, `develop`, and `release/[release name]` branches are up-to-date.
- [x] Checkout the `release/[release name]` branch.
- [x] Run `git flow release finish [release name]`
- [x] Run `git push --follow-tags`
- [x] Test the site.

## After release
- [x] Review changes on production
- [x] Alert #fec-partners of the successful release

**Comment by jenniferthibault** [comment]
@ccostino @LindsayYoung: should we check off the items above and close?

**Comment by ccostino** [comment]
@jenniferthibault Thanks for the reminder!  I just ran through it to make sure all is done and removed a couple of extraneous steps that are no longer needed (they were focused on the old web app deployments).

Closing this issue. :-)
